bots-deployment-detector/package.json

```
@@ -1,10 +1,10 @@
 {
-  "name": "forta-agent-starter",
+  "name": "nethermind-bots-deployment-detector",
   "version": "0.0.1",
-  "description": "Forta Agent Typescript starter project",
+  "description": "This Forta Bot detects any transactions that creates/updates a bot from Nethermind.",
   "repository": "https://github.com/forta-network/forta-bot-sdk/tree/master/starter-project",
```

```suggestion
  "repository": "https://github.com/Macbeth98/forta-bots/tree/master/bots-deployment-detector",
```
this is the repository where the bot will be pushed into